ksqldb-engine/src/test/java/io/confluent/ksql/engine/KsqlEngineTest.java

```diff
@@ -981,27 +981,21 @@ public void shouldNotThrowWhenPreparingDuplicateTable() {
   public void shouldThrowWhenExecutingDuplicateTable() {
     // Given:
     final List<ParsedStatement> parsed = ksqlEngine.parse(
-        "CREATE TABLE FOO AS SELECT * FROM TEST2; "
+        "CREATE TABLE FOO WITH (KAFKA_TOPIC='BAR') AS SELECT * FROM TEST2; "
```

Comment:
This change should be reverted.

Reply:
This is a new test now that verifies the `IF NOT EXISTS` works.